### ISSUE TITLE
 Call to undefined method BeyondCode\LaravelMaskedDumper\DumpSchema::__set_state()

### ISSUE THREAD

**Issue by gggolddeity** [beyondcode/laravel-masked-db-dump#21]
Getting this error on config:cache execution

Using dump schema file:

class DumpSchema extends \BeyondCode\LaravelMaskedDumper\DumpSchema
{
    /**
     * @throws Exception
     */
    public function getConnection(): ConnectionInterface
    {
        $connection = parent::getConnection();
        $connection->getDoctrineSchemaManager()->getDatabasePlatform()->registerDoctrineTypeMapping('enum', 'string');

        return $connection;
    }
}

**Comment by neilgilmour** [comment]
It's not in the docs but I found this by searching in this repo;

https://github.com/beyondcode/laravel-masked-db-dump/pull/15#issue-1353665464

tldr: put the definition into a helper and put the helper in the config
